litmus-portal/cluster-agents/subscriber/pkg/k8s/operations.go

```diff
@@ -158,52 +160,42 @@ func IsClusterConfirmed() (bool, string, error) {
 
 // ClusterRegister function creates litmus-portal config map in the litmus namespace
 func ClusterRegister(clusterData map[string]string) (bool, error) {
-	clientset, err := GetGenericK8sClient()
+	clientset, err:= GetGenericK8sClient()
 	if err != nil {
 		return false, err
 	}
 
-	newConfigMapData := map[string]string{
-		"IS_CLUSTER_CONFIRMED": clusterData["IS_CLUSTER_CONFIRMED"],
-		"SERVER_ADDR":          clusterData["SERVER_ADDR"],
-		"AGENT_SCOPE":          clusterData["AGENT_SCOPE"],
-		"COMPONENTS":           clusterData["COMPONENTS"],
-		"START_TIME":           clusterData["START_TIME"],
-		"VERSION":              clusterData["VERSION"],
-		"SKIP_SSL_VERIFY":      clusterData["SKIP_SSL_VERIFY"],
-		"CUSTOM_TLS_CERT":      clusterData["CUSTOM_TLS_CERT"],
+	configMapPatch:= map[string]interface{}{
+		"data": clusterData,
+	}
+	configMapPatchOutput, err:= json.Marshal(configMapPatch)
+	if err != nil {
+		return false, err
 	}
 
-	_, err = clientset.CoreV1().ConfigMaps(AgentNamespace).Update(context.TODO(), &corev1.ConfigMap{
-		Data: newConfigMapData,
-		ObjectMeta: metav1.ObjectMeta{
-			Name: AgentConfigName,
-		},
-	}, metav1.UpdateOptions{})
+	_, err = clientset.CoreV1().ConfigMaps(AgentNamespace).Patch(context.TODO(),AgentConfigName,types.MergePatchType,configMapPatchOutput,metav1.PatchOptions{},)
 	if err != nil {
 		return false, err
 	}
 
-	logrus.Info(AgentConfigName + " has been updated")
+	logrus.Info(AgentConfigName+" has been updated")
```

Comment:
```suggestion
	logrus.Infof("%s has been patched", AgentConfigName)
```

Reply:
Sure Ill makes these changes and update the PR